packages/react/src/components/List/List.story.jsx

```diff
@@ -620,3 +620,100 @@ export const WithPagination = () => (
 );
 
 WithPagination.storyName = 'with pagination';
+
+export const WithLoadMore = () => {
+  const ListWithExpandIds = () => {
+    const [expandedIds, setExpandedIds] = useState([]);
+    const [loadingMoreIds, setLoadingMoreIds] = useState([]);
+
+    const [listItems, setListItems] = useState([
+      {
+        id: 'org',
+        content: { value: 'Organization' },
+        children: [
+          { id: 'site-01', content: { value: 'Site 1' } },
+          {
+            id: 'site-02',
+            content: { value: 'Site 2' },
+            children: [
+              { id: 'system-01', content: { value: 'System 1' } },
+              { id: 'system-02', content: { value: 'System 2' } },
+            ],
+            hasLoadMore: true,
+          },
+          {
+            id: 'site-03',
+            content: { value: 'Site 3' },
+            children: [
+              { id: 'system-03', content: { value: 'System 3' } },
+              { id: 'system-04', content: { value: 'System 4' } },
+            ],
+            hasLoadMore: true,
+          },
+        ],
+      },
+    ]);
+    return (
+      <div style={{ width: 400 }}>
+        <List
+          title="Sports Teams"
+          iconPosition="left"
+          items={listItems}
+          expandedIds={expandedIds}
+          toggleExpansion={(id) => setExpandedIds((prev) => [...prev, id])}
```

Comment:
In order to have the storybook work appropriately we should probably add a way to close parent.
```suggestion
          toggleExpansion={(id) => setExpandedIds((prev) => prev.includes(id) ? prev.filter((prevId) => prevId !== id) : [...prev, id])}
```